cpp/src/Slice/Grammar.y

```diff
@@ -1929,22 +1929,23 @@ type
     TypeList types = cont->lookupType(scoped->v);
     if (types.empty())
     {
-        YYERROR; // Can't continue, jump to next yyerrok
+        $$ = nullptr;
```

Comment:
`YYERROR` is a Bison macro which is basically 'throwing a syntax error'.
`$$ = ` is how you set a return value in Bison.

So, instead of throwing a syntax error, we just return `nullptr` now.